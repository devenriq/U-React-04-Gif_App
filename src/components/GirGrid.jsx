import React from "react";

export const GirGrid = ({ category }) => {
  return (
    <>
      <h3>{category}</h3>
      <p>Hola mundo</p>
    </>
  );
};
