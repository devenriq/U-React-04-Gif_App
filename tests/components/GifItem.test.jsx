import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe("Pruebas en GifItem component", () => {
  const title = "Saitama";
  const url = "https://picsum.photos/200";

  test("should match with snapshot", () => {
    const { container } = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
});
