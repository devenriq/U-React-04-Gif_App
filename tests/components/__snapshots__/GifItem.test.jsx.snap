// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifItem component should match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Saitama"
      src="https://picsum.photos/200"
    />
    <p>
      Saitama
    </p>
  </div>
</div>
`;
